### ISSUE TITLE
`Schema::dropIfExists()` not working in 11.2.2+

### ISSUE THREAD

**Issue by staudenmeir** [yajra/laravel-oci8#853]
### Summary of problem or feature request

Hi @yajra,
`Schema::dropIfExists()` no longer works for me since version `11.2.2`. The table doesn't get dropped when it exists.

I traced it to this change: https://github.com/yajra/laravel-oci8/commit/dfebaf8bd59781983372de1cbc071af370105e52

### Code snippet of problem

```php
Schema::dropIfExists('users');
```

Executed SQL in `11.2.1`:

```sql
declare
    c int;
begin
    select count(*) into c from user_tables where table_name = upper('users');
    if c = 1 then
        execute immediate 'drop table users';
    end if;
end;
```

Executed SQL in `11.2.2`:

```sql
declare
    c int;
begin
    select count(*) into c from user_tables where table_name = upper('"USERS"');
    if c = 1 then
        execute immediate 'drop table "USERS"';
    end if;
end;
```

The issue is this part where the table name gets wrapped in two pairs of quotes:

> upper('"USERS"')

### System details

- Operating System: Ubuntu
- PHP Version: `8.3.7`
- Laravel Version: `11.7.0`
- Laravel-OCI8 Version: `11.2.2` (and `11.2.3`)

**Comment by yajra** [comment]
Fixed in v11.2.4. thanks for reporting. 🍻

**Comment by staudenmeir** [comment]
Thanks!
